pdfly/cli.py

```diff
@@ -37,6 +38,29 @@ def version_callback(value: bool) -> None:
     ),
     rich_markup_mode="rich",  # Allows to pretty-print commands documentation
 )
+@entry_point.command(name="sign", help="Sign a PDF file using a PKCS12 (.p12) certificate.")
+def sign(
+    input_pdf: Path = typer.Argument(..., help="Input PDF file."),
+    output_pdf: Path = typer.Option(..., "--output", "-o", help="Output signed PDF."),
+    p12: Optional[Path] = typer.Option(None, "--p12", help="PKCS12 certificate file."),
+    password: Optional[str] = typer.Option(None, "--password", help="Password for the certificate."),
+) -> None:
+    """
+    Sign a PDF file using a PKCS12 (.p12) certificate.
+    """
+    print(f"input_pdf: {input_pdf}")
+    print(f"output_pdf: {output_pdf}")
+    print(f"p12: {p12}")
+    print(f"password: {password}")
```

Comment:
This is fine for debugging, but those `print()` calls will have to be removed in the final version of this PR 🙂